src/host-handler.js

```diff
@@ -204,7 +177,7 @@ var hostHandler = {
                     }
                 });
                 $.ajax({
-                    url: 'https://' + location.hostname +
+                    url: 'http://' + location.hostname +
```

Comment:
It looks like mediathread-chrome had an outdated version of mediathread-collect. These links should be https, not http. I'm going to change them back to https here.